backend/schema/corpora_schema.md

```diff
@@ -1,125 +1,71 @@
-# Corpora Schema
+# cellxgene Data Integration Schema
 
 Authors: mkinsella@chanzuckerberg.com
 
 Document Status: _Approved_
 
-Version: 1.0.0
+Version: 1.1.0
 
-Date Last Modified: 2020-07-21
+Date Last Modified: 2020-10-26
 
 
 ## Background
 
-Corpora is for "publishing, exploring, integrating, and re-using" single-cell datasets. There is a tension in those goals. On one hand, focusing on
-publication and exploration means Corpora should let submitters be in full control of how their datasets appear on the platform. This includes everything 
-from metadata fields to the colors in their plots. On the other hand, focusing on integration and re-use means Corpora should require datasets to follow a
-standard schema even if that's at odds with submitters' preferences. Complicating this further, there is no current consensus on exactly how to annotate single
-cell data, and any schema that Corpora adopts will evolve significantly over time.
+cellxgene aims to support the publication and sharing of single-cell datasets as well as the construction of a data corpus that facilitates data integration
+across multiple tissues and experiments. Achieving the latter goal requires some harmonization of metadata and features in the cellxgene Data Portal. But if
+that harmonization is too onerous, it will burden the first goal of rapid data sharing.
```

Comment:
Is this "first goal" as in the one listed first above? Or as in first in priority? I would suggest trimming to
> ...it will burden the goal of rapid data sharing.

in order to remove the possible confusion.

This line also seems unnecessary for the audience of contributors, but more appropriate for internal justification.

Reply:
...reading further, it seems this may not be aimed at contributors. Can you clarify the intended audience for this?

Reply:
Will make that change. The audience for the doc is a bit mixed, but I thought a contributor might want to see some brief justification for the level of detail in the schema.